sites/partners/cypress/e2e/03-listing.spec.ts

```diff
@@ -256,10 +256,15 @@ describe("Listing Management Tests", () => {
       cy.getByTestId("ag-search-input").type(listing["name"])
       cy.getByTestId(listing["name"]).first().click()
       cy.getByTestId("listingEditButton").contains("Edit").click()
-      cy.getByTestId("nameField").should("be.visible").click().type(" (Edited)")
+      cy.getByTestId("nameField")
+        .should("be.visible")
+        .click()
+        .clear()
+        .clear()
```

Comment:
do we need the double .clear() and if so is there maybe a better way to set this up so that we aren't running into race conditions

Reply:
unfortunately no :( its been a problem for us consistently 
I did find: https://filiphric.com/how-to-wait-for-page-to-load-in-cypress which may have some options for us to try. 

I think what leaps out to me is perhaps waiting for the api call that gets the data since I think once we get the data react should be a relatively fast rerender?

Reply:
The weird thing about this spot is that we aren't waiting for an api call for this field. The field exists and has data, we are just typing values on it. Since I'm unable to reproduce locally it has also been hard to test

Reply:
ah, yeah maybe we can spin up a spike to investigate this further, I would love if our cypress tests could be less race-condition-y 

I'll give this pr a 👍 thought since it looks good and solves the problem

ty ty

Reply:
Sounds good! I'll create a ticket with all of our known cypress flakiness